src/store/messagesStore.js

```diff
@@ -796,6 +798,15 @@ const actions = {
 
 		context.commit('loadedMessagesOfConversation', { token })
 
+		if (invisibleMessages > 95) {
+			// If there are more than 95 invisible messages we load another chunk
```

Comment:
`95` seems to be a magic number for me here. If I understand correctly, that is because default limit of chunk is 100 messages and the problem occurs when a whole chunk consists only of "invisible messages". Currently web client never sets the limit and always uses default, so limit does not occur on client side.

What about adding a constant like `DEFAULT_FETCH_MESSAGES_LIMIT`?

Reply:
> and the problem occurs when a whole chunk consists only of "invisible messages"

Not only, but yeah then you get the "numb" feeling. But also if it loads for a long time and then only one message shows and you have to scroll down and up again it is weird.

Reply:
Oh, I see now. So then I'd prefer to add 2 constants for this place to be clean.
Something like `DEFAULT_FETCH_MESSAGE_LIMIT = 100` and `MIN_PREFERED_FETCH_VISIBLE_MESSAGES = 5`.
Or at least a comment with description / link to issue, so it will be no magic in 95...

> if it loads for a long time and then only one message shows and you have to scroll down and up again it is weird

Maybe then require 20-25 messages instead of 5? It is approximate amount of messages takes up the whole 4K screen forcing user to scroll up for previous messages.

Reply:
There is also a hard but strict way - fetch messages and count visible messages continuously until  there will be 100 visible messages or messages ends. (Or even do this part on server side). But not sure it's worth it =D

Reply:
Made a change that should improve the situation